src/index.html

```diff
@@ -4,10 +4,37 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
```

Comment:
```suggestion
    <article class="card" data-qa="card">
```
Consider using semantic tags for the content